TestCases/rans/naca0012/turb_NACA0012_sst_expliciteuler.cfg

```
@@ -0,0 +1,196 @@
+%
+% Mach number (non-dimensional, based on the free-stream values)
+MACH_NUMBER= 0.15
+%
+% Angle of attack (degrees, only for compressible flows)
+AOA= 10.0
+%
+% Free-stream temperature (288.15 K by default)
+FREESTREAM_TEMPERATURE= 300.0
+%
+% Reynolds number (non-dimensional, based on the free-stream values)
+REYNOLDS_NUMBER= 6.0E6
+%
+% Reynolds length (1 m by default)
+REYNOLDS_LENGTH= 1.0
+%
+% ---------------------- REFERENCE VALUE DEFINITION ---------------------------%
+%
+% Reference origin for moment computation
+REF_ORIGIN_MOMENT_X = 0.25
+REF_ORIGIN_MOMENT_Y = 0.00
+REF_ORIGIN_MOMENT_Z = 0.00
+%
+% Reference length for pitching, rolling, and yawing non-dimensional moment
+REF_LENGTH= 1.0
+%
+% Reference area for force coefficients (0 implies automatic calculation)
+REF_AREA= 1.0
+%
+% Flow non-dimensionalization (DIMENSIONAL, FREESTREAM_PRESS_EQ_ONE,
+%                              FREESTREAM_VEL_EQ_MACH, FREESTREAM_VEL_EQ_ONE)
+REF_DIMENSIONALIZATION= FREESTREAM_PRESS_EQ_ONE
+
+% -------------------- BOUNDARY CONDITION DEFINITION --------------------------%
+%
+% Navier-Stokes wall boundary marker(s) (NONE = no marker)
+MARKER_HEATFLUX= ( airfoil, 0.0 )
+%
+% Farfield boundary marker(s) (NONE = no marker)
+MARKER_FAR= ( farfield )
+%
+% Marker(s) of the surface to be plotted or designed
+MARKER_PLOTTING= ( airfoil )
+%
+% Marker(s) of the surface where the functional (Cd, Cl, etc.) will be evaluated
+MARKER_MONITORING= ( airfoil )
+
+% ------------- COMMON PARAMETERS DEFINING THE NUMERICAL METHOD ---------------%
+%
+% Numerical method for spatial gradients (GREEN_GAUSS, WEIGHTED_LEAST_SQUARES)
+NUM_METHOD_GRAD= WEIGHTED_LEAST_SQUARES
+NUM_METHOD_GRAD_RECON= LEAST_SQUARES
+%
+% Courant-Friedrichs-Lewy condition of the finest grid
+CFL_NUMBER= 0.1
+%
+% Max Delta time
+MAX_DELTA_TIME= 1E10
+%
+% Number of total iterations
+ITER= 99999
+
+% ------------------------ LINEAR SOLVER DEFINITION ---------------------------%
+%
+% Linear solver or smoother for implicit formulations (BCGSTAB, FGMRES, SMOOTHER)
+LINEAR_SOLVER= FGMRES
+%
+% Preconditioner of the Krylov linear solver (ILU, LU_SGS, LINELET, JACOBI)
+LINEAR_SOLVER_PREC= ILU
+%
+% Minimum error of the linear solver for implicit formulations
+LINEAR_SOLVER_ERROR= 1E-10
+%
+% Max number of iterations of the linear solver for the implicit formulation
+LINEAR_SOLVER_ITER= 20
+
+% -------------------- FLOW NUMERICAL METHOD DEFINITION -----------------------%
+%
+% Convective numerical method (JST, LAX-FRIEDRICH, CUSP, ROE, AUSM, HLLC,
+%                              TURKEL_PREC, MSW)
+CONV_NUM_METHOD_FLOW= ROE
+USE_VECTORIZATION= YES
+%
+% Spatial numerical order integration (1ST_ORDER, 2ND_ORDER, 2ND_ORDER_LIMITER)
+MUSCL_FLOW= YES
+%
+% Slope limiter (VENKATAKRISHNAN, MINMOD)
+SLOPE_LIMITER_FLOW= NONE
+%
+% Time discretization (RUNGE-KUTTA_EXPLICIT, EULER_IMPLICIT, EULER_EXPLICIT)
+TIME_DISCRE_FLOW= EULER_EXPLICIT
```

If you have a minute can you remove the comments from the options please? Together with options that are not active.
A lot of this stuff is out of date from being copied across all SU2 versions.
```suggestion
CONV_NUM_METHOD_FLOW= ROE
USE_VECTORIZATION= YES
MUSCL_FLOW= YES
SLOPE_LIMITER_FLOW= NONE
TIME_DISCRE_FLOW= EULER_EXPLICIT
```

---

I removed the option desciptions in `turb_NACA0012_sst_expliciteuler.cfg` and `turb_NACA0012_sst_fixedvalues.cfg`. The latter has been generated from `turb_NACA0012_sst.cfg`  in #1236 by throwing out the most unnecessary stuff. I don't know which of the remaining options could and should be removed.

---

Looks clean enough :+1: